README.md

```
@@ -121,11 +121,18 @@ If you cannot log in or see video feeds, double-check that your `.env` file matc
+**Note: This project has migrated from pip to uv for dependency management.**
+
 To install Python packages required for development, run:
 
 ```sh
-pip install -r requirements.txt
-pip install -r requirements-dev.txt
+uv sync --group dev
+```
+
+Legacy pip installation (deprecated):
+```sh
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)